scripts/algcomptool/AlgCompTool.py

```diff
@@ -0,0 +1,335 @@
+#Author: Mihai Avram - mihai.v.avram@gmail.com
+#Date: 3/15/2017
+
+import sys
+import argparse
+import pandas as pd
+import numpy as np
+import string
+
+def main():
+	parser = argparse.ArgumentParser()
+	#Required parameters -a1 and -a2 which denote the paths to the two input algorithm files to be compared (print output to the console)
+	parser.add_argument("-a1", "--alg1file", type=str, help="the relative or absolute location and name of the input file of the first algorithm", required=True)
```

Comment:
You can just pass `add_argument("alg1file", type=str...)`and that will become a positional argument. Positional arguments are required. Also, I recommend using more descriptive names for the user, e.g. `input-file-1`; if you still want the destination to be called `alg1file` you may pass `dest="alg1file"` so that you don't need to change the rest of your code.